src/services/charts/useReposCoverageMeasurements.ts

```diff
@@ -40,15 +44,17 @@ const query = `
     $isPublic: Boolean
   ) {
     owner(username: $owner) {
-      measurements(
```

Comment:
TODO - remove this use case - owner.measurements is different than owner.repo.measurements (the one i'm after)